includes/Dashboard/Templates/Products.php

```diff
@@ -387,6 +387,7 @@ public function handle_product_update() {
 
         $errors     = array();
         $post_title = sanitize_text_field( $postdata['post_title'] );
+        $post_slug  = isset( $postdata['editable-post-name'] ) && ! empty( $postdata['editable-post-name'] ) ? sanitize_text_field( $postdata['editable-post-name'] ) : '';
```

Comment:
use sanitize_title() instead of sanitize_text_field()

Reply:
This is normal text field where have slug, this is not title, so we should use sanitize_text_field

Reply:
sanitize_title is used to sanitize slug